apps/cli-daemon/src/app/workspace/workspace.service.ts

```diff
@@ -23,11 +23,17 @@ import {
   NOT_ANGULAR_WORKSPACE_EXCEPTION,
 } from './entities';
 
+export const ng = 'npx ng new';
```

Comment:
rename to "NEW_COMMAND"

Reply:
and create commands.ts file to store all this consts

Reply:
done